packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -151,9 +164,20 @@ export class DatabricksDriver extends JDBCDriver {
       azureKey:
         conf?.azureKey ||
         getEnv('dbExportBucketAzureKey', { dataSource }),
+
+      dbCatalog: conf?.dbCatalog || getEnv('databricksDbCatalog', { dataSource }),
+      databricksStorageCredentialName: conf?.databricksStorageCredentialName || getEnv('databricksStorageCredentialName', { dataSource }),
     };
+
     super(config);
     this.config = config;
+
+    if (config.dbCatalog) {
+      this.preaggregationSchema = getEnv('preAggregationsSchema') ||
```

Comment:
📓 unfortunately we can't build regular expression which will be deterministic in terms of parsing because user can use SELECT on different catalogs in his own cube. 

e.g.

```
cube(`smth`, {
   SQL:  `SELECT * FROM samples.tpch.lineitem LIMIT 1000`
}
```

So without knowing at least one part of the search string it's impossible to build such kind of expression